tests/test_api_v1.py

```diff
@@ -126,16 +130,25 @@ def test_fetch_paginated_requests(mock_chain, app, auth_env, client, db):
     assert len(fetched_requests) == 20
     for repo_number, request in enumerate(fetched_requests):
         assert request['repo'] == repo_template.format(repo_number)
+    assert response['meta']['previous'] is None
+    assert fetched_requests[0]['dependencies'] == 13
 
     # per_page and page parameters are honored
-    rv = client.get('/api/v1/requests?page=2&per_page=10')
+    rv = client.get('/api/v1/requests?page=2&per_page=10&verbose=True')
     assert rv.status_code == 200
     response = json.loads(rv.data.decode('utf-8'))
     fetched_requests = response['items']
     assert len(fetched_requests) == 10
     # Start at 10 because each page contains 10 items and we're processing the second page
     for repo_number, request in enumerate(fetched_requests, 10):
         assert request['repo'] == repo_template.format(repo_number)
+    pagination_metadata = response['meta']
+    for page, page_num in [('next', 3), ('last', 5), ('previous', 1), ('first', 1)]:
+        assert 'page={}'.format(page_num) in pagination_metadata[page]
+        assert 'per_page=10' and 'verbose=True' in pagination_metadata[page]
```

Comment:
This assert statement isn't checking if `'per_page=10'` is in `pagination_metadata[page]`. It's just checking if that's truthy. How about the following?
```python
assert 'per_page=10' in pagination_metadata[page]
assert 'verbose=True' in pagination_metadata[page]
```